inc/compat/themes/compat-theme-leadengine.php

```diff
@@ -55,11 +61,33 @@ public function change_sticky_elements_relative_header( $attributes ) {
 		// Bail if using distraction free header and footer
 		if ( FluidCheckout_CheckoutPageTemplate::instance()->is_distraction_free_header_footer_checkout() ) { return $attributes; }
 
-		$attributes['data-sticky-relative-to'] = '.navbar-default';
+		$attributes[ 'data-sticky-relative-to' ] = '.navbar-default';
 
 		return $attributes;
 	}
 
+
+
+	/**
+	 * Add button class from the theme.
+	 * 
+	 * @param  array|string  $classes  The button classes.
+	 */
+	public function add_button_class( $classes ) {
+		// Define button class
+		$button_class = 'tt_button';
+
+		// Add button class to the classes array
+		if ( is_array( $classes ) ) {
+			array_push( $classes, $button_class );
+		}
+		// Otherwise append button class as a string
+		else {
+			$classes .= ' ' . $button_class;
+		}
+
+		return $classes;
+	}
```

Comment:
@luiggidev, this is nice how you reuse the function for various hooks, and treat `array` and `string` parameters. I made a couple changes to set the button class name only once, and added a code comment for what happens inside `if` and `else`.